llvm/test/SYCLLowerIR/wg_scope_ctor_loop.ll

```diff
@@ -0,0 +1,65 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py
+; RUN: opt < %s -LowerWGScope -S | FileCheck %s
+
```

Comment:
NIT: would be nice to add the C++ source of the SYCL kernel used to generate this IR as a comment.